tensorflow/python/ops/parallel_for/pfor.py

```diff
@@ -1593,7 +1593,7 @@ def _add_to_stack(x):
           else:
             converter = _pfor_converter_registry.get(y_op.type, None)
           if converter is None:
-            root_cause = (f"there is no register converter for this op.")
+            root_cause = (f"there is no registered converter for this op yes.")
```

Comment:
"yet"

Reply:
Actually we prefer to avoid words such as "yet" which suggest a promise of future work. So please just remove "yes"/"yet".

Reply:
Ok